Apps/Inc/SendDisplay.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __SEND_DISPLAY_H
+#define __SEND_DISPLAY_H
+
```

Comment:
Why would you not just include these in the source file?

Reply:
Fixed